dev/integration_tests/ios_platform_view_tests/ios/PlatformViewUITests/PlatformViewUITests.m

```diff
@@ -27,18 +29,22 @@ - (void)setUp {
   [self.app launch];
 }
 - (void)testPlatformViewFocus {
-
-  XCUIElement *entranceButton = self.app.buttons[@"platform view focus test"];
-  XCTAssertTrue([entranceButton waitForExistenceWithTimeout:1]);
-  [entranceButton tap];
+  [self waitForAndTapElement:self.app.buttons[@"platform view focus test"]];
+  /// Tries to wait and tap the button the second time, if the first time failed.
+  /// https://github.com/flutter/flutter/pull/90535
+  BOOL newPageAppeared = [self.app.textFields[@"platform_view[0]"] waitForExistenceWithTimeout:kStandardTimeOut];
+  if (!newPageAppeared) {
+    NSLog(@"First try failed. The tree is %@", self.app.debugDescription);
+    [self waitForAndTapElement:self.app.buttons[@"platform view focus test"]];
```

Comment:
Retapping seems like overkill at this point since we don't know what the tree looks like. Just increase the timeout and log on failure, then we'll have more information about whether there's a real bug, or if we need workarounds like retapping.

Reply:
Sounds good. Updated